docs/v2/full-text/boost.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Boosting
+---
+
+Boosting increases or decreases the impact of a specific query by multiplying its contribution to the overall BM25 score.
+
+## Constant Boosting
+
+To boost a query, cast the query to the `boost` type. In this example, the `shoes` query is weighted twice as heavily as the `footwear` query.
+
+```sql
+SELECT id, paradedb.score(id), description, category
+FROM mock_items
+WHERE description ||| 'shoes'::boost(2) OR category ||| 'footwear'
+ORDER BY score DESC
+LIMIT 5;
+```
+
+`boost` takes a numeric value, which is the multiplicative boost factor. It can be any floating point number between `-2048` and `2048`.
```

Comment:
AFAICT, this must be an integer: it looks like it is internally converted to a float.

@eeeebbbbrrrr: Which makes me wonder: would decreasing the range here be easier to reason about? Maybe allowing between -100 and 100 or something?

Reply:
I’m not following?  Tantivy’s boost is a f32. 

Unfortunately we only have room for a f16 in this typmod stuff

Reply:
I guess I just don't know the use cases, and how much precision they actually require. Is -100 to 100 (200 values) "enough" distinct boost values? I don't know whether it's usually automation choosing the boost values, or humans.